### PR TITLE
Surpress pull_request closed

### PR DESCRIPTION
**Pull request by TooAngel**

### CODE REVIEW

**Comment by TooAngel** [comment]
DCBOT: This repository is under [democratic collaboration](https://github.com/TooAngel/democratic-collaboration) and will be merged automatically.

**Comment by TooAngel** [comment]
DCBOT: Code update recognized, countdown starts fresh.

**Comment by Nevrdid** [comment]
test on global comments

File: src/server.py [hunk 1]
```diff
@@ -54,9 +54,10 @@ def execute_edited(self):
         print(self.data['changes'])
 
     def execute_closed(self):
-        # TODO check PR and add message that this is under voting
-        print(self.data)
-        print(self.data.keys())
+        # TODO Anything to do here?
```
Comment (Nevrdid) [inline src/server.py]: Dunno

Comment (Nevrdid) [inline src/server.py]: a review here ?

File: src/server.py [hunk 2]
```diff
@@ -71,6 +72,10 @@ def handle_pull_request_review(self, data):
         if data['action'] == 'submitted':
             if data['review']['state'] == 'commented':
                 # TODO take as last point for countdown
```
Comment (Nevrdid) [inline src/server.py]: new review

**Comment by TooAngel** [comment]
DCBOT: Code update recognized, countdown starts fresh.

**Pull request merged**